daos/story-dao.ts

```diff
@@ -79,16 +84,18 @@ export default class StoryDao implements StoryDaoI {
    * database
    */
   findStories = async (): Promise<Story[]> => {
-    return StoryModel.find().populate("postedBy");
+    return StoryModel.find().populate("viewers").populate("postedBy").exec();
 }
 
   /**
    * Retrieves all stories by visibility, as stories can be visible to public, friends or
    * close friends groups
-   * @param visibility Visibility of story
+   * @param uid
    */
-  findStoriesByVisibility = async (visibility: string): Promise<Story[]> => {
-    return StoryModel.find({visibility: visibility}).populate("postedBy");
+  findStoriesVisibleToUser = async (uid: string): Promise<Story[]> => {
+    const stories = await this.findStories();
+    const user = await StoryDao.userDao.findUserById(uid);
```

Comment:
We are not using this `user` instance anywhere. Can we remove it?

Reply:
yeap! removed